guide/src/building_and_distribution.md

```diff
@@ -230,6 +234,14 @@ When cross-compiling, PyO3's build script cannot execute the target Python inter
 * `PYO3_CROSS_PYTHON_VERSION`: Major and minor version (e.g. 3.9) of the target Python installation. This variable is only needed if PyO3 cannot determine the version to target from `abi3-py3*` features, or if `PYO3_CROSS_LIB_DIR` is not set, or if there are multiple versions of Python present in `PYO3_CROSS_LIB_DIR`.
 * `PYO3_CROSS_PYTHON_IMPLEMENTATION`: Python implementation name ("CPython" or "PyPy") of the target Python installation. CPython is assumed by default when this variable is not set, unless `PYO3_CROSS_LIB_DIR` is set for a Unix-like target and PyO3 can get the interpreter configuration from `_sysconfigdata*.py`.
 
+An experimental `pyo3` crate feature `generate-abi3-import-lib` enables the user to cross compile
+"abi3" extension modules for Windows targets without setting `PYO3_CROSS_LIB_DIR` environment
+variable or providing any Windows Python library files. It uses external `python3-dll-a` crate
```

Comment:
A link to the crate homepage might be a nice courtesy?

Reply:
I'll add a link to `docs.rs` page. As for the GitHub project page, I actually hoped to transfer that project to the PyO3 organization after this feature is done. Unfortunately, there's a chance I might lose access to my GitHub or Gmail accounts because of the circumstances, and I don't want PyO3 to end up with an unmaintainable dependency.

Reply:
I've added links to the `docs.rs` page to both the user guide and Architecture.md.

Reply:
The changes look fine to me. Leaving this unresolved due to the question of long-term maintenance though.

Reply:
I don't want to be misunderstood here. I _do_ intend to maintain it and to contribute to PyO3 in the future, but I also want to safeguard against the situation where I'd be unable to do so. GitHub has blocked accounts of users from sanctioned countries before, and if that happens I don't want it to become a problem for the projects that depend on mine. That's why I want to transfer the GitHub project and the `crates.io` package ownership to an organization as a backup measure.